AEPAnalytics/Sources/AnalyticsHelper.swift

```diff
@@ -27,6 +32,17 @@ class AnalyticsHelper {
         }
         return ret
     }
+
+    /// Combines the application name, version, and version code into a formatted application identifier
+    /// - Returns: `String` formatted Application identifier
+    static func getApplicationIdentifier() -> String {
+        let applicationName = systemInfoService.getApplicationName() ?? ""
+        let applicationVersion = systemInfoService.getApplicationVersionNumber() ?? ""
+        let applicationBuildNumber = systemInfoService.getApplicationBuildNumber() ?? ""
+        // Make sure that the formatted identifier removes white space if any of the values are empty, and remove the () version wrapper if version is empty as well
+        return "\(applicationName) \(applicationVersion) (\(applicationBuildNumber))".replacingOccurrences(of: "  ", with: " ").replacingOccurrences(of: "()", with: "").trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
for this code: 
```swift
.replacingOccurrences(of: "  ", with: " ")
```
do we want the replacement to be `""` instead?

Reply:
If applicationVersion is empty, we still need `" "` in between.